windows-nsis/build-snapshot

```diff
@@ -31,7 +32,7 @@ main() {
 	git clone -q --depth 1 --branch "${OPENVPN_BRANCH}" "${OPENVPN_URL}" || die "openvpn.clone"
 	cd openvpn
 	autoreconf -ivf || die "autoreconf.openvpn"
-	./configure --disable-lzo --disable-snappy --disable-lz4 || die "openvpn.configure"
+	./configure --disable-lzo --disable-lz4 || die "openvpn.configure"
```

Comment:
ok, I will add another PR for release/2.3 branch, probably. or you can do it yourself.